Gemfile

```diff
@@ -1,6 +1,6 @@
 source 'https://rubygems.org'
 
-ruby "2.2.3"
+ruby "2.3.0"
```

Comment:
[Rails 5 requires Ruby 2.2.2+](http://edgeguides.rubyonrails.org/upgrading_ruby_on_rails.html#ruby-2-2-2). We will need to find a way to signpost that in for devs. I agree that the way it's implemented in gemfiles etc is poor, but without it you end up locking in CI configs etc which is also non optimal.